autotest/tools/pipeline/test_pipeline_chat_pytorch.py

```diff
@@ -25,12 +25,13 @@ def test_pipeline_chat_pytorch_tp1(config, common_case_config, model,
     if 'gw' in worker_id:
         os.environ['CUDA_VISIBLE_DEVICES'] = get_cuda_id_by_workerid(worker_id)
     p = Process(target=run_pipeline_chat_test,
-                args=(config, common_case_config, model, 'pytorch'))
+                args=(config, common_case_config, model, 'pytorch', worker_id))
```

Comment:
what's the motivation for introducing `work_id`?

Reply:
> what's the motivation for introducing `work_id`?

When executing testcases of pipeline in parallel, testcase verification of the same model may conflict so the testcase will fail. woker_id in pytest can tell me which worker process is used, I can solve this bug.